test/installer/scripts/omsadmin_systest.rb

```diff
@@ -296,4 +310,28 @@ def test_reconstruct_from_partial_empty_omsadmin_conf
                  "tmp directory was mistakenly created")
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/, lso, "The command to show what should be the default proc limit did not yield an integer.")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso = show_omsagent_proc_limit()
+    assert_match(/^5000$/, lso, "Should have been 5000 process limit.")
+  end
+
+  def test_show_omsagent_proc_limit_two_entries
+    FileUtils.cp("#{@omsadmin_test_dir}/limits-two-settings.conf", @proc_limits_conf)
+    lso = show_omsagent_proc_limit()
+    assert_match(/^100$/, lso, "Multiple entries should still report last entry, in compliance with specification.")
```

Comment:
Add a line to check that the "2 $TEST_USER entries found" message is printed.

Reply:
Sorry I missed this Friday.  Looking now.